jax_galsim/__init__.py

```diff
@@ -0,0 +1,9 @@
+from jax.config import config
+
+config.update("jax_enable_x64", True)
```

Comment:
yep exactly, so I initially started off without this option, but then some of the tests for the Gaussian profile were not passing, at like 7th decimal ^^ so I don't think it's super critical. 
But with this option enabled I think this will match the choices made by galsim, which is prbably a good default to have

Reply:
the things that may break are convolutions, but most likely we will be doing them using Fourier anyway, so ....

Reply:
> But with this option enabled I think this will match the choices made by galsim, which is prbably a good default to have

I agree, will this though make JAX slower? Or mostly the same ? 

> the things that may break are convolutions, but most likely we will be doing them using Fourier anyway, so ....

Yup! No reason to do them in real space, at least that I can think of

Reply:
> and include in the doc some comments telling the user how to explicitly control numerical precision... which is actually what jax does
That might be preferable to avoid the regression in performance if it is unnecessary for an unsuspecting user... A user who cares about very precise results will care about these issues and read the docs. 

I think you mentioned that some tests were failing without this, could we enable this setting just for those tests ?

Reply:
we can definitely let the users decide for themselves, and enable it by default for tests.

Reply:
Ok, I've removed it for now, the failiing tests are not yet running, they depend on some other components.

I propose we add this back when and where necessary, and we should maybe open a dedicated issue to discuss it when the problem comes back. I think ensuring the validity  of outputs is more important than speed, so there is something to be said about enforcing float64 as default

Reply:
that's a good point! I'm OK opening an issue to discuss it when it comes back